### PR TITLE
[5.8] Adjust issue & pr templates

### PR DESCRIPTION
**Pull request by driesvints**
Minor adjustments to the issue and pr templates to clarify supported versions.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_Bug_report.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 name: "🐛 Bug Report"
-about: Report a general framework issue
+about: 'Report a general framework issue. Please first check if your Laravel version is still supported: https://laravel.com/docs/5.8/releases#support-policy'
```
Comment (staudenmeir) [inline .github/ISSUE_TEMPLATE/1_Bug_report.md]: Always use the latest version:

```suggestion
about: 'Report a general framework issue. Please first check if your Laravel version is still supported: https://laravel.com/docs/releases#support-policy'
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
 <!--
+Please only send in a PR to branches which are currently supported: https://laravel.com/docs/5.8/releases#support-policy
```
Comment (staudenmeir) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Please only send in a PR to branches which are currently supported: https://laravel.com/docs/releases#support-policy 
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
 <!--
+Please only send in a PR to branches which are currently supported: https://laravel.com/docs/5.8/releases#support-policy 
+
+If you are unsure to which branch you want to send a PR please read: https://laravel.com/docs/5.8/contributions#which-branch
```
Comment (staudenmeir) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
If you are unsure to which branch you want to send a PR please read: https://laravel.com/docs/contributions#which-branch
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (driesvints) [reply .github/PULL_REQUEST_TEMPLATE.md]: The docs don't support page forwards with anchor tags.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (driesvints) [reply .github/PULL_REQUEST_TEMPLATE.md]: The docs don't support page forwards with anchor tags.

File: .github/ISSUE_TEMPLATE/1_Bug_report.md (hunk 1 above)
Comment (driesvints) [reply .github/ISSUE_TEMPLATE/1_Bug_report.md]: The docs don't support page forwards with anchor tags.

Comment (driesvints) [reply .github/ISSUE_TEMPLATE/1_Bug_report.md]: Hmm, for some reason this works now. weird

Comment (driesvints) [reply .github/ISSUE_TEMPLATE/1_Bug_report.md]: Pushed. Thanks!

**Pull request merged**